package.json

```diff
@@ -3,8 +3,16 @@
   "version": "0.1.0",
   "description": "Retrieve updates based on activities happening across the GitHub organization",
   "main": "index.js",
+  "source": "index.js",
+  "files": [
+    "index.js",
+    "sources",
+    "targets",
+    "README.md",
+    "LICENSE"
```

Comment:
I think these get included anyways, gotta double check

Reply:
i think you're right

```shell
% npm pack --dry-run
npm notice 
npm notice 📦  @eqworks/updates@0.1.0
npm notice === Tarball Contents === 
npm notice 1.1kB  LICENSE          
npm notice 12.6kB sources/github.js
npm notice 151B   index.js         
npm notice 332B   targets/slack.js 
npm notice 732B   package.json     
npm notice 48B    README.md        
npm notice === Tarball Details === 
npm notice name:          @eqworks/updates                        
npm notice version:       0.1.0                                   
npm notice filename:      eqworks-updates-0.1.0.tgz               
npm notice package size:  5.1 kB                                  
npm notice unpacked size: 15.0 kB                                 
npm notice shasum:        44bbede0356974985d77e62f39cf47142b6ada14
npm notice integrity:     sha512-spr7+Rb77ULGf[...]D+tVhT+onsKFw==
npm notice total files:   6                                       
npm notice 
eqworks-updates-0.1.0.tgz
```